Features/Standard/platformui.feature

```diff
@@ -0,0 +1,132 @@
+Feature: Login on PlatformUI
+
+    @javascript
+    Scenario: As a admin User, I want to login to PlatformUI
+        Given I go to PlatformUI app
+        And I fill in "Username" with "admin"
+        And I fill in "Password" with "publish"
+        When I click on the "Login" button
+        Then I should see "Welcome to eZ Platform" title
+
+    @javascript
+    Scenario: As a admin User, I want to login to PlatformUI
+        Given I go to PlatformUI app with username "admin" and password "publish"
+        Then I should see "Welcome to eZ Platform" title
+
+    @javascript
+    Scenario: As a admin User, I want to see Activity tab
+        Given I go to PlatformUI app with username "admin" and password "publish"
+        And I click on "Root location" link
+        When I click on the tab "Activity"
+        Then I should see "Activity content" text
+
+    @javascript
+    Scenario: As a admin User, I want to use side menu
+        Given I go to PlatformUI app with username "admin" and password "publish"
+        And I click on "Root location" link
+        When I click on the logo
+        Then I should see "Welcome to eZ Platform" title
+
+    @javascript
+    Scenario: As a admin User, I want to see navigate menu
+        Given I go to PlatformUI app with username "admin" and password "publish"
+        And I click on the navigation zone "Platform"
+        When I clicked on the "Media library" link
+        Then I should see "Media" title
+
+    @javascript
+    Scenario: As a admin User, I want to see navigate menu
+        Given I go to PlatformUI app with username "admin" and password "publish"
+        And I click on the navigation zone "Admin Panel"
+        When I click on the navigation zone "Platform"
+        And I clicked on the "Media library" link
+        Then I should see "Media" title
+
+    @javascript
+    Scenario: As a admin User, I want to see navigate menu
+        Given I go to PlatformUI app with username "admin" and password "publish"
+        And I click on the navigation zone "Admin Panel"
+        When I click on the "Administration dashboard" link
+        Then I should see "Administration dashboard" title
+
+    @javascript
+    Scenario: As a admin User, I want to see navigate menu
+        Given I go to PlatformUI app with username "admin" and password "publish"
+        And I click on the menu zone "Create"
+        When I click on the sub-menu option "Insite editing"
```

Comment:
the *Insite editing* menu is gone for quite while now and was hardcoded like the activity tab to manually tests some behaviour.